src/firefly/js/visualize/WebPlot.js

```diff
@@ -260,22 +261,22 @@ export const PlotAttribute= {
 
 const relatedIdRoot= '-Related-';
 
-export const isHiPS= (plot) => Boolean(plot && plot.type==='hips');
-export const isImage= (plot) => Boolean(plot && plot.type==='image');
-export const isKnownType= (plot) => Boolean(plot && (plot.type==='image' || plot.type==='hips'));
+export const isHiPS= (plot) => Boolean(plot && plot.plotType==='hips');
+export const isImage= (plot) => Boolean(plot && plot.plotType==='image');
+export const isKnownType= (plot) => Boolean(plot && (plot.plotType==='image' || plot.plotType==='hips'));
 
 /**
  *
  * @param plotId
- * @param type
+ * @param plotType
  * @param asOverlay
  * @param imageCoordSys
  * @return {WebPlot}
  */
-function makePlotTemplate(plotId, type, asOverlay, imageCoordSys) {
+function makePlotTemplate(plotId, plotType, asOverlay, imageCoordSys) {
     return {
         plotId,
-        type,
+        plotType,
```

Comment:
in file 'CsysConverter.js' constructor, there is line like 'this.type = plot.type'. Should the same change be applied to that line? I found that marker rendering is invisible and the problem is because there is no 'plotType' key in CsysConverter object.

Reply:
yes! thank you for catching that.  The was a very hard refactor.